### DISCUSSION TITLE
6.0.0-beta.1

### DISCUSSION THREAD

**Discussion by dennisdoomen** [fluentassertions/fluentassertions#1595] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/fluentassertions/fluentassertions/releases/tag/6.0.0-beta.1'>6.0.0-beta.1</a>.</em>

**Comment by eNeRGy164** [comment]
Is there any work to pick up for the beta?

* Tested with `LivingDocumentation`, only issue was the removed `.BeEquivalentTo(params object[])`.
  * The release notes do not link to the upgrade documentation
* Tested with `dapr/dotnet-sdk`, works without change
* Tested with `PlantUML.Builder`, works without change

**Reply by dennisdoomen** [reply]
* Dogfooding
* [Bugs](https://github.com/fluentassertions/fluentassertions/issues?q=is%3Aopen+is%3Aissue+label%3Abug)

**Reply by jnyrup** [reply]
The removal of `BeEquivalentTo(params object[])` is mentioned in https://github.com/fluentassertions/fluentassertions/pull/1529/files#diff-d888d040691f3300c2f347c3dd1fac50fb6f166ae0ea18073cbc640371350632R180
But let's add it to release notes also to make the change more visible.

**Reply by eNeRGy164** [reply]
> The removal of `BeEquivalentTo(params object[])` is mentioned in https://github.com/fluentassertions/fluentassertions/pull/1529/files#diff-d888d040691f3300c2f347c3dd1fac50fb6f166ae0ea18073cbc640371350632R180
> But let's add it to release notes also to make the change more visible.

I did find it, but more by chance. I would expect the release notes to at least link to the upgrade documentation

**Reply by jnyrup** [reply]
Ohh, I did mention the removal in the release notes. 
https://github.com/fluentassertions/fluentassertions/pull/1529/files#diff-66bff3ed7b10da4ff2003461ce1c5a22828c018dfde1a22ef20d9c19a961936cR30-R32

Are we looking at different release notes?
https://github.com/fluentassertions/fluentassertions/blob/develop/docs/_pages/releases.md

**Reply by eNeRGy164** [reply]
Nowhere on the release notes page does it link to https://github.com/fluentassertions/fluentassertions/blob/develop/docs/_pages/upgradingtov6.md

As a consumer of FA, my code breaks after upgrading. So, I can see it in the _breaking changes_ section, but I'm not helped on _how to fix_ the issue.
That _is_ mentioned in the upgrade page, but you have to know that page exists.

**Reply by dennisdoomen** [reply]
Yeah, we should link to that page from the release notes.
